### PR TITLE
LPS-173958 [Site Initializer] Change _addOrUpdateObjectFields from JSONObject to JSONArray

### PR DESCRIPTION
**Pull request by joseabelenda**

### CODE REVIEW

**Comment by joseabelenda** [comment]
ci:test:sf

**Comment by joseabelenda** [comment]
ci:forward

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle/src/main/resources/site-initializer/object-fields/test-object-fields-1.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+	"object-field": [
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle/src/main/resources/site-initializer/object-fields/test-object-fields-1.json:2]: "object-field" should be "object-fields"

**Review by nNilton — CHANGES_REQUESTED** [review]

**Comment by joseabelenda** [comment]
resent:https://github.com/pt-liferay-solutions/liferay-portal/pull/3226

**Pull request closed without merging**